compose/ui/ui-uikit/src/uikitMain/objc/CMPUIKitUtils/CMPUIKitUtils/CMPOSLoggerInterval.m

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "CMPOSLoggerInterval.h"
+
+#import <os/signpost.h>
+
+@implementation CMPOSLoggerInterval {
+    os_log_t _log;
+    os_signpost_id_t _signpostId;
+    NSString *_name;
```

Comment:
Do we use this field? Let's remove if we don't use.

Reply:
Yes, indeed. Removed.